website/content/docs/job-specification/group.mdx

```diff
@@ -329,6 +329,46 @@ group "second" {
 }
 ```
 
+#### Known Limitations
+
+The `max_client_disconnect` configuration is a new operating model for Nomad. Due
+to the nature of the change and the challenges it does and does not address, there
+are some known limitations to the feature that are detailed below.
```

Comment:
This language becomes stale almost immediately, and at least the first two behaviors we're describing aren't going to change. I'd just leave this section and the top header out.